src/thriveopedia/pages/ThriveopediaHomePage.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Godot;
+
+/// <summary>
+///   Thriveopedia page displaying welcome information and links to websites.
+/// </summary>
+public class ThriveopediaHomePage : ThriveopediaPage
+{
+    public override string PageName => "Home";
+    public override string TranslatedPageName => TranslationServer.Translate("HOME_PAGE");
```

Comment:
I think the thriveopedia pages should have "THRIVEOPEDIA_" prefix, and maybe "_TITLE" suffix for all of the page names for super clarity.